### PR TITLE
tests/drivers_*: use bus parameters from board or driver

### PR DESCRIPTION
**Pull request by PeterKietzmann**
Some sensors define default parameters in their drivers, especially I'm talking about the bus- and/or bus-addresses. Sensors are permanently mounted on a board or externally connected via cables. In the first case, the default parameters are also defined in the board.h which makes sense. Most of the sensor tests define these parameters by itself which may lead to configuration errors. In addition to that, some I2C devices have address pins to set up a different bus address. That shows the need of separation between (i) board-default and (ii) sensor **or** personal configuration. 

Question 1:
What do you think about it and my proposal for the `tmp006` in this PR? 

Question 2:
Do you think the driver itself should define a default value (which may change according to address pins) or should we rather define the default in the test application and leave it to the user in case there isn't a default configuration e.g. from the board.h?

### CODE REVIEW

File: drivers/include/tmp006.h [hunk 1]
```diff
@@ -84,8 +84,8 @@ extern "C"
 {
 #endif
 
-#ifndef TMP006_I2C_ADDRESS
-#define TMP006_I2C_ADDRESS         0x41 /**< TMP006 Sensor Default Address */
+#ifndef TMP006_ADDR
+#define TMP006_ADDR                0x40 /**< TMP006 Sensor Default Address */
```
Comment (jfischer-phytec-iot) [inline drivers/include/tmp006.h]: Why is the default state of the pin C1 and B1 low ? or why you changed the address?

**Comment by jfischer-phytec-iot** [comment]
Do we still need the tests/driver_... for sensors, maybe we should/can do it better with SAUL?
(TMP006 was already saulized in #4155)

File: drivers/include/tmp006.h (hunk 1 above)
Comment (PeterKietzmann) [inline drivers/include/tmp006.h]: The board.h itself defines the address and thus the address pin setting. On phyNODEs one pin is high and one is low which leads to I2C address 0x41 (IIRC). However, using e.g. an Atmel board with the tmp006 adafruit sensor breakout board I prefer to define the "default" as "not connected". IIRC the sensor breakout board has pull-down resistors on it for the address lines.

**Comment by PeterKietzmann** [comment]
I personally like separate, minimal sensor tests but I don't feel like to decide this

**Comment by OlegHahm** [comment]
Have your feelings changed? ;-)

**Comment by PeterKietzmann** [comment]
No they didn't. However, this PR won't bring a decision IMO. Guess we need to discuss remotely or so... Closing with memo label set.

**Comment by haukepetersen** [comment]
This issue should be solved with the introduction of default parameters via `xxx_param.h` files anyway, right?!

**Comment by PeterKietzmann** [comment]
Yes issue should be solved. The question about superseding separate driver applications with one saul application is not answered.

**Pull request closed without merging**